sencha-workspace/pages/src/page/TeacherCompetencyDashboard.js

```diff
@@ -29,21 +30,24 @@ Ext.define('Site.page.TeacherCompetencyDashboard', {
         // initialize QuickTips
         Ext.QuickTips.init();
 
-
         // empty content editor container
         teacherDashboardCt.empty();
 
-
         // render teacher dashboard component
         me.dashboard = dashboard = Ext.create('Slate.cbl.view.teacher.Dashboard', {
             renderTo: teacherDashboardCt,
             contentArea: contentAreaData
         });
 
-
         // load data embedded in page
-        Ext.getStore('cbl-students-loaded').loadRawData(studentsData);
+        studentsDataStore = Ext.getStore('cbl-students-loaded');
+
+        studentsDataStore.sorters.add(new Ext.util.Sorter({
```

Comment:
Actually this would be better specified as a `sorters` config within the declaration of the store model